lib/responses.test.js

```diff
@@ -410,4 +410,35 @@ describe(_.startCase(filename), function () {
       fn()(req, res);
     });
   });
+
+  describe('checkArchivedToPublish', function () {
+    const fn = lib[this.title];
+
+    it('throws an error if checkProps returns true', function () {
+      const req = {
+          body: { archived: true },
+          uri: 'domain.com/_pages/id'
+        },
+        res = createMockRes();
+
+      db.getMeta.returns(Promise.resolve({ published: true }));
+
+      expectStatus(res, 400);
+      fn(req, res);
+    });
+
+    it('calls next if checkProps returns false', function () {
+      let req = {
+        body: {archived: true},
+        uri: 'domain.com/_pages/id'
+      };
+      const res = createMockRes(),
+        next = sinon.stub();
+
+      db.getMeta.returns(Promise.resolve({published: false}));
```

Comment:
Remember to add spaces in here
```suggestion
      db.getMeta.returns(Promise.resolve({ published: false }));
```